src/api/config.rs

```diff
@@ -109,10 +110,9 @@ pub struct EncoderConfig {
   pub tiles: usize,
   /// Number of frames to read ahead for the RDO lookahead computation.
   pub rdo_lookahead_frames: usize,
-  /// If enabled, computes the PSNR values and stores them in [`Packet`].
-  ///
-  /// [`Packet`]: struct.Packet.html#structfield.psnr
-  pub show_psnr: bool,
+  /// Contains metrics such as PSNR, SSIM, etc. which may be enabled
+  /// by CLI flags.
+  pub metrics_enabled: MetricsEnabled,
```

Comment:
the encoder and the CLI are completely separated entities, the field description should not refer to the CLI.